packages/p2p/src/utils/websocket.js

```diff
@@ -133,3 +139,9 @@ export const requestWS = async (request) => {
 
     return modified_response;
 };
+
+export const subscribeWS = (request, cb) => {
+    ws.p2pSubscribe(request, (response) => {
```

Comment:
p2pSubscribe will return something, can we pls make it return for this also ?

Reply:
oh, I thought you were making that change. should I make it?

Reply:
that was my workaround 😅only. we can change that later, its not breaking anything